Kube_events_test is flaky

```
ok  	k8s.io/heapster/sinks/influxdb	0.021s
E0911 10:19:27.005150    3197 iowatcher.go:108] Unable to decode an event from the watch stream: got invalid watch event type: 
--- FAIL: TestEventsBasic-2 (0.01s)
	Location:	kube_events_test.go:43
	Error:		No error is expected but got Error: received an event list with nil Items.
		
FAIL
FAIL	k8s.io/heapster/sources	0.072s
```

--- a/sources/kube_events_test.go
+++ b/sources/kube_events_test.go
@@ -28,6 +28,7 @@ import (
 )
 
 func TestEventsBasic(t *testing.T) {
+	t.Skip("Broken and flaky test - first fix #537 and #565")
 	handler := util.FakeHandler{
 		StatusCode:   200,
 		RequestBody:  "something",

